### PR TITLE
Chore: release notes for 2020.16

### LINKED ISSUE #5242: release note 2020.16
release note 2020.16

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #5242 

**Additional issues**:

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,80 @@ id: Release notes
...
+- [@patternfly/react-catalog-view-extension@4.8.124](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/4.8.124)
+- [@patternfly/react-charts@6.12.10](https://www.npmjs.com/package/@patternfly/react-charts/v/6.12.10)
+- [@patternfly/react-core@4.84.2](https://www.npmjs.com/package/@patternfly/react-core/v/4.84.2)
+- [@patternfly/react-icons@4.7.22](https://www.npmjs.com/package/@patternfly/react-icons/v/4.7.22)
+- [@patternfly/react-inline-edit-extension@4.5.185](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/4.5.185)
+- [@patternfly/react-styles@4.7.21](https://www.npmjs.com/package/@patternfly/react-styles/v/4.7.21)
+- [@patternfly/react-table@4.19.43](https://www.npmjs.com/package/@patternfly/react-table/v/4.19.43)
+- [@patternfly/react-tokens@4.9.18](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.9.18)
+- [@patternfly/react-topology@4.6.88](https://www.npmjs.com/package/@patternfly/react-topology/v/4.6.88)
+- [@patternfly/react-virtualized-extension@4.5.169](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.5.169)
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:16]: These versions will need to change once we've wrapped up the charts changes.

**Review by redallen — CHANGES_REQUESTED** [review]

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,80 @@ id: Release notes
...
+- [@patternfly/react-core@4.84.3](https://www.npmjs.com/package/@patternfly/react-core/v/4.84.3)
+- [@patternfly/react-icons@4.7.22](https://www.npmjs.com/package/@patternfly/react-icons/v/4.7.22)
+- [@patternfly/react-inline-edit-extension@4.5.186](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/4.5.186)
+- [@patternfly/react-styles@4.7.22](https://www.npmjs.com/package/@patternfly/react-styles/v/4.7.22)
+- [@patternfly/react-table@4.19.44](https://www.npmjs.com/package/@patternfly/react-table/v/4.19.44)
+- [@patternfly/react-tokens@4.9.22](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.9.22)
+- [@patternfly/react-topology@4.6.89](https://www.npmjs.com/package/@patternfly/react-topology/v/4.6.89)
+- [@patternfly/react-virtualized-extension@4.5.170](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.5.170)
+
+- **Alert:**
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:18]: Add `## Components` heading

**Review by redallen — APPROVED** [review]
Just needs this one fix then LGTM!

**Review by evwilkin — APPROVED** [review]
LGTM

**Review by jschuler — APPROVED** [review]

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -56,21 +56,13 @@ section: developer-resources
 - **Tree view:** 
   - Fixed various bugs ([#5128](https://github.com/patternfly/patternfly-react/pull/5128))
 
-### Catalog view extension
-- Fixed typings in FilterSidePanelCategoryItem ([#5145](https://github.com/patternfly/patternfly-react/pull/5145))
-
-### Access console extension
-- Removed css imports ([#5147](https://github.com/patternfly/patternfly-react/pull/5147))
-- Replaced novnc-core usage with the original @novnc/novnc ([#5208](https://github.com/patternfly/patternfly-react/pull/5208))
```
Comment (mcoker) [inline packages/react-docs/RELEASE-NOTES.md]: Just want to verify this shouldn't be in the notes?

Comment (tlabaj) [reply packages/react-docs/RELEASE-NOTES.md]: You are right!  I deleted one too many line.

**Review by mcoker — APPROVED** [review]
🥳

**Review by evwilkin — APPROVED** [review]
LGTM - thanks!

**Pull request merged**